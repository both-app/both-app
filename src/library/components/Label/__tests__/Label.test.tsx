import React from 'react'
import { render } from 'react-native-testing-library'

import { Label } from '../'

describe('Label', () => {
  test('should display the label', () => {
    const { toJSON } = render(
      <Label primary="primary text" secondary="secondary text" />
    )

    expect(toJSON()).toMatchSnapshot()
  })

  test('should not display the secondary text', () => {
    const { toJSON } = render(<Label primary="primary text" />)

    expect(toJSON()).toMatchSnapshot()
  })
})
