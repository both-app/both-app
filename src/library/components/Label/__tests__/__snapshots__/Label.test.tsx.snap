// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should display the label 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#0C2333",
        "fontFamily": "DMSerifDisplay-Regular",
        "fontSize": 26,
      }
    }
  >
    primary text
  </Text>
  <Text
    style={
      Object {
        "color": "#0C2333",
        "fontFamily": "DMSerifDisplay-Regular",
        "fontSize": 26,
      }
    }
  >
    secondary text
  </Text>
</View>
`;

exports[`Label should not display the secondary text 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#0C2333",
        "fontFamily": "DMSerifDisplay-Regular",
        "fontSize": 26,
      }
    }
  >
    primary text
  </Text>
</View>
`;
